### DISCUSSION TITLE
How to invert the filter?

### DISCUSSION THREAD

**Discussion by seghers** [jarun/nnn#1896] [Q&A] [answered]
Is there a way to invert the filter?

I'd like to show all file in the folder except those containing the word `foobar`

I tried using regex filter (pressing / 2x)
```^((?!foobar).)*$```
but it's not working

**Comment by jarun** [comment, marked as answer]
I see it working with PCRE: https://github.com/jarun/nnn/wiki/Developer-guides#compile-with-pcre
 
Perhaps the default POSIX regex has limitations. I see basic matches working.

**Reply by seghers** [reply]
I think I'm using it with pcre using a nix override.
Can nnn show if pcre is enabled?

**Reply by seghers** [reply]
It works with PCRE, it didn't work with the default Posix regex

**Reply by N-R-K** [reply]
> Can nnn show if pcre is enabled?

You can use `lddtree` or `ldd` to see what dynamic libraries are being linked against.
